client/src/components/InternshipCard.jsx

```diff
@@ -1,12 +1,20 @@
 import React from "react";
+import { Box } from "@mui/material";
 
 /**
  * https://github.com/BobaTalks/internship-tracker/issues/25
  * Component for the internship card. should take an Internship as props
  *
  */
 const InternshipCard = () => {
-  return <></>;
+  return (
+    <Box
+      bgcolor="gray.50"
+      borderRadius="2rem"
+      padding="2rem"
+      sx={{ boxShadow: "0px 3px 3px 0px #00000033" }}
```

Comment:
is there an out of the box component from MUI we can use here?

Reply:
I think Card is a great option to start, and if we're thinking larger picture, if we do need to override, we can ask ourselves id the Card or Cards across the app will be mostly the same and establish base style overrides at the theme level